pubtools/_quay/operator_pusher.py

```diff
@@ -414,7 +414,7 @@ def build_index_images(self):
         return iib_results
 
     @log_step("Push index images to Quay")
-    def push_index_images(self, iib_results):
+    def push_index_images(self, iib_results, timestamp=None):
```

Comment:
Missing docstring

Reply:
Also, I feel like this name is too tied to the current implementation. I would maybe rename it to "extra_tag" or "redundant_tag" or something. I think calling it timestamp conveys information that's irrelevant in this context.